modules/data.atmosphere/R/align_met.R

```diff
@@ -0,0 +1,298 @@
+##' Align meteorology datasets for debiasing
+# -----------------------------------
+# Description
+# -----------------------------------
+##'
+##' @title align.met
+##' @family debias - Debias & Align Meteorology Datasets into continuous time series
+##' @author Christy Rollinson
+##' @description This script aligns meteorology datasets in at temporal resolution for debiasing & 
+##'              temporal downscaling.  
+##'              Note: The output here is stored in memory!  
+##'              Note: can probably at borrow from or adapt align_data.R in Benchmarking module, but 
+##'              it's too much of a black box at the moment.
+# -----------------------------------
+# Parameters
+# -----------------------------------
+##' @param train.path - path to the dataset to be used to downscale the data
+##' @param source.path - data to be bias-corrected aligned with training data (from align.met)
+##' @param yrs.train - (optional) specify a specific years to be loaded for the training data; 
+##'                    prevents needing to load the entire dataset.  If NULL, all available years 
+##'                    will be loaded. If not null, should be a vector of numbers (so you can skip
+##'                    problematic years)
+##' @param n.ens  - number of ensemble members to generate and save
```

Comment:
It would be good to eventually merge align.data and align.met in a single functions that aligns any two time series. But understand the limitation on that for now.